homeassistant/components/velbus/__init__.py

```diff
@@ -58,6 +58,21 @@ async def velbus_scan_task(
         raise PlatformNotReady(
             f"Connection error while connecting to Velbus {entry_id}: {ex}"
         ) from ex
+    # create all modules
+    dev_reg = dr.async_get(hass)
+    for module in controller.get_modules().values():
+        dev_reg.async_get_or_create(
+            config_entry_id=entry_id,
+            identifiers={
+                (DOMAIN, str(module.get_addresses()[0])),
```

Comment:
Could the api change order of addresses? If so this could be problematic. Maybe add all addresses instead?

Reply:
A module always has a master address (element 0 in the list). Some modules can have up to 5 subadresses, they are not ordered and start from element 1 in the list.

So element 0 will always be the master address.

Reply:
Ok, sounds good.